plugin/README.md

```diff
@@ -22,6 +22,10 @@ Plugin inputs can be set in `netlify.toml`. They're all optional.
   - `feat/*`: matches all branches starting with `feat/`
   - `*-bug`: matches all branches finishing with `-bug`
 - `disabled` - _Default: `false`_ - Use to disable the plugin without removing it.
+- `pathPrefix` - The prefix of your website if it's not at the root level.
+  Putting "pathPrefix: /blog" will alias `/blog` to `/`
+- `customDomain` - The custom domain that you use, if it's not possible to define it on your Netlify's settings.
```

Comment:
Same here, it's not exactly that they can't define it in Netlify, but more a problem with hardcoded urls no?

Reply:
the underlying issue yes, but we are exposing this property because not all sites can define a custom domain in the Netlify settings, aka the property here is solving the lack of settings. I will add a comment about the initial issue